app/models/project.rb

```diff
@@ -1,4 +1,12 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+
+  def average_experience
+    total_experience = contestants.sum(&:years_of_experience)
```

Comment:
If you find yourself doing a lot of calculations with Ruby, always try to stop and think about what AR methods might exist. This is more Ruby than we'd like to see. Try this instead in a pry and see what you get! `.average(:years_of_experience)`